{{cookiecutter.project_shortname}}/Pipfile.lock

```diff
@@ -205,14 +211,16 @@
                 "sha256:7546cc08e3899716e12fe67d12d7cfe9a64647014d1134b014c3c392b63cad42",
                 "sha256:aada5cfdc4a543c47098eb3aca6663848ef5d04b4324935ced441debc11ec98b"
             ],
+            "markers": "extra == 'elasticsearch6'",
             "version": "==6.3.1"
         },
         "elasticsearch-dsl": {
             "hashes": [
-                "sha256:794fc7e03fc580cf48c35c9491fa97f08afa1b9e897f9c59dfa285efb520d71c",
-                "sha256:d0a5137caf43c2ecb69f7a867d52d40e557b3ba16c361dff74a07affe7101c38"
+                "sha256:5114a38a88e93a4663782eae07a1e8084ba333c49887335c83de8b8043bc72b2",
+                "sha256:d6d974cd2289543a3350690494a43fe9996485b8dc6f1d8758cb56bee01244bd"
             ],
-            "version": "==6.2.1"
+            "markers": "extra == 'elasticsearch6'",
+            "version": "==6.1.0"
```

Comment:
Is there a reason why we use 6.1.0 instead of 6.2.1?

Reply:
I have removed the `Pipfile.lock`, I think it should be responsibility of the services to generate their `Pipfile.lock` for the production instances. Also @egabancho mentioned it in previous PR's https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/74..